packages/tspd/package.json

```
@@ -73,8 +73,8 @@
     "mocha": "~10.2.0",
     "source-map-support": "~0.5.19",
     "rimraf": "~5.0.1",
-    "typedoc-plugin-markdown": "~4.0.0-next.18",
-    "typedoc": "~0.24.8",
+    "typedoc-plugin-markdown": "~4.0.0-next.20",
+    "typedoc": "~0.25.1",
```

```suggestion
    "typedoc-plugin-markdown": "4.0.0-next.20",
    "typedoc": "0.25.1",
```

All pre-release (`0.Y.Z` or `X.Y.Z-foo`) versions should be pinned rather than floated (even using `~`), since semver allows breaking changes between any pre-release versions.